extension/src/WorkspaceFunctions.ts

```diff
@@ -75,6 +75,23 @@ export function getTranslationFolderPath(ResourceUri?: vscode.Uri) {
     return translationFolderPath;
 }
 
+export function getDtsWorkFolderPath(ResourceUri?: vscode.Uri) {
+    let workspaceFolder = getWorkspaceFolder(ResourceUri);
+    let workspaceFolderPath = workspaceFolder.uri.fsPath;
+    let dtdWorkFolderPath = path.join(workspaceFolderPath, '.dts');
+    return dtdWorkFolderPath;
```

Comment:
Maybe this is a bit of a stretch but the declaration of `workspaceFolder` seems unnecessary. If this function is changed, same changes should be made in the function `getTranslationFolderPath` :)
```suggestion
    return path.join(getWorkspaceFolder(ResourceUri).uri.fsPath, '.dts');
```